nuget/determine-pr-version-base.yml

```diff
@@ -0,0 +1,77 @@
+parameters:
+  manualTriggerVersion: ''
+  packageVersionVariableName: 'PackageVersion'
+  pullRequestFormat:
+    type: string
+    default: PR
+    values:
+     - PR
+     - dev
+  invidualCIFormat:
```

Comment:
Should be `individualCIFormat`?

Reply:
![](https://media.giphy.com/media/5xtDarlq6nrviGVugXm/source.gif)